dev-java/openjfx/openjfx-11.0.11_p1-r1.ebuild

```diff
@@ -0,0 +1,226 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV="${PV/_p/+}"
+SLOT="${MY_PV%%[.+]*}"
+EGRADLE_VER="4.10.3"
+
+inherit flag-o-matic java-pkg-2 multiprocessing
+
+DESCRIPTION="Java OpenJFX client application platform"
+HOMEPAGE="https://openjfx.io"
+
+SRC_URI="https://hg.openjdk.java.net/${PN}/${SLOT}-dev/rt/archive/${MY_PV}.tar.bz2 -> ${P}.tar.bz2
+	https://downloads.gradle.org/distributions/gradle-${EGRADLE_VER}-bin.zip
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-sandbox/7.1.0/lucene-sandbox-7.1.0.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-grouping/7.1.0/lucene-grouping-7.1.0.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-queryparser/7.1.0/lucene-queryparser-7.1.0.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-queries/7.1.0/lucene-queries-7.1.0.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-core/7.1.0/lucene-core-7.1.0.jar
+	https://repo.maven.apache.org/maven2/org/antlr/gunit/3.5.2/gunit-3.5.2.jar
+	https://repo1.maven.org/maven2/org/antlr/antlr4/4.7.2/antlr4-4.7.2-complete.jar
+	https://repo.maven.apache.org/maven2/org/antlr/ST4/4.0.8/ST4-4.0.8.jar
+"
+
+LICENSE="GPL-2-with-classpath-exception"
+KEYWORDS="-* ~amd64 ~ppc64"
+
+IUSE="cpu_flags_x86_sse2 debug doc source +media"
+
+RDEPEND="
+	dev-java/swt:4.10[cairo,opengl]
+	dev-libs/atk
+	dev-libs/glib:2
+	dev-libs/libxml2:2
+	dev-libs/libxslt
+	media-libs/freetype:2
+	media-libs/fontconfig:1.0
+	x11-libs/gdk-pixbuf
+	x11-libs/gtk+:2
+	x11-libs/gtk+:3
+	x11-libs/cairo[glib]
+	x11-libs/libX11
+	x11-libs/libXtst
+	x11-libs/libXxf86vm
+	x11-libs/pango
+	virtual/jpeg
+	virtual/opengl
```

Comment:
You may want to depend on `app-accessibility/at-spi2-core` and `media-libs/libjpeg-turbo` directly, instead of `dev-libs/atk` and `virtual/jpeg`.